man/sysupdate.d.xml

```diff
@@ -487,6 +487,42 @@
         <xi:include href="version-info.xml" xpointer="v251"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ChangeLog=</varname></term>
+
+        <listitem><para>Optionally takes a human-presentable URL to a website containing a change-log of
+        the resource being updated.</para>
+
+        <para>At most one transfer definition (one <filename>*.conf</filename> file in the directory)
+        can include this setting. Thus, the website should describe all changes for all the transfer
+        definitions. For instance, in the foobarOS example above, any URL specified here should describe
+        changes in both the kernel UKI and in the rootfs.</para>
+
+        <para>This setting supports specifier expansion. See below for details on supported
+        specifiers. This setting will also expand the <literal>@v</literal> wildcard pattern. See above
```

Comment:
In an unresolved discussion on the other PR (https://github.com/systemd/systemd/pull/32363#discussion_r1591540207), Lennart suggests that you make `AppStream=` support `@v` too.

I'm not sure that we should do that. AppStream metadata is supposed to be version-agnostic. Specifically, it will have changelogs for multiple versions in it. So ultimately I don't think it makes sense for there to be multiple appstream catalogs separated by `@v`. This is why I didn't initially make `AppStream=` support the `@v` specifier

Reply:
Agreed, AppStream is supposed to handle multiple versions, so I don't think it makes sense to allow version expansion for it.